error: no rules expected the token `}`
 --> tests/ui/pin_project/unsupported.rs:3:1
  |
3 | / pin_project! {
4 | |     struct Struct1 {} //~ ERROR no rules expected the token `}`
5 | | }
  | |_^ no rules expected this token in macro call
  |
  = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `}`
 --> tests/ui/pin_project/unsupported.rs:3:1
  |
3 | / pin_project! {
4 | |     struct Struct1 {} //~ ERROR no rules expected the token `}`
5 | | }
  | |_^ no rules expected this token in macro call
  |
  = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `(`
 --> tests/ui/pin_project/unsupported.rs:8:19
  |
8 |     struct Struct2(); //~ ERROR no rules expected the token `(`
  |                   ^ no rules expected this token in macro call

error: no rules expected the token `;`
  --> tests/ui/pin_project/unsupported.rs:12:19
   |
12 |     struct Struct3; //~ ERROR no rules expected the token `;`
   |                   ^ no rules expected this token in macro call

error: no rules expected the token `enum`
  --> tests/ui/pin_project/unsupported.rs:15:1
   |
15 | / pin_project! {
16 | |     enum Enum { //~ ERROR no rules expected the token `enum`
17 | |         A(u8)
18 | |     }
19 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `enum`
  --> tests/ui/pin_project/unsupported.rs:15:1
   |
15 | / pin_project! {
16 | |     enum Enum { //~ ERROR no rules expected the token `enum`
17 | |         A(u8)
18 | |     }
19 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `union`
  --> tests/ui/pin_project/unsupported.rs:21:1
   |
21 | / pin_project! {
22 | |     union Union { //~ ERROR no rules expected the token `union`
23 | |         x: u8,
24 | |     }
25 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `union`
  --> tests/ui/pin_project/unsupported.rs:21:1
   |
21 | / pin_project! {
22 | |     union Union { //~ ERROR no rules expected the token `union`
23 | |         x: u8,
24 | |     }
25 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::__pin_project_expand` (in Nightly builds, run with -Z macro-backtrace for more info)
